### ISSUE TITLE
Accessibility: staff interface (/control) doesn't have a lang attribute on the html element

### ISSUE THREAD

**Issue by sandbergja** [subjectsplus/SubjectsPlus#1371]
The public view of guides does have a lang attribute, but the staff interface does not.  This is a WCAG Level A violation: https://webaim.org/standards/wcag/checklist#sc3.1.1
